plugins/modules/system/sap_task_list_execute.py

```diff
@@ -0,0 +1,437 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sap_task_list_execute
+short_description: Perform SAP Task list execution.
+version_added: "3.5.0"
+description:
+  - The C(sap_task_list_execute) module will perform RFC execution on SAP Systems
+    for tasks lists.
+  - Tasks in the task list which requires manual activities will be confirmed automatically.
+  - This module will use the RFC Package C(STC_TM_API).
+
+options:
+  state:
+    description:
+      - If the state C(show) is provided
+        The possible parameters are shown for each task.
+        The returned values are in the necessary format for C(task_parameters).
+      - The full list of parameters is exported to stdout in json format.
+    choices: ['show', 'present']
+    required: false
+    default: present
+    type: str
+  conn_username:
+    description: The required username for the SAP system.
+    required: true
+    type: str
+  conn_password:
+    description: The required password for the SAP system.
+    required: true
+    type: str
+  host:
+    description: The required host for the SAP system. Can be either an FQDN or IP Address.
+    required: true
+    type: str
+  sysnr:
+    description:
+      - The system number of the SAP system.
+      - This value must be single quoted. Otherwise ansible treated this like a number and replace leading zeros.
+      - Defaults to C('01')
+    required: false
+    default: '00'
+    type: str
+  client:
+    description:
+      - The client number to connect to.
+      - This value must be single quoted. Otherwise ansible treated this like a number and replace leading zeros.
+      - Defaults to C('001')
+    default : '000'
+    required: false
+    type: str
+  task_to_execute:
+    description: The task list which will be executed.
+    required: true
+    type: str
+  task_parameters:
+    description:
+      - The tasks and the parameters for execution.
+      - If the task list do not need any parameters. This could be empty.
+      - The list values must have brackets (see example).
+      - If only specific tasks from the task list should be executed.
+        The tasks even when no parameter is needed must be provided.
+        Alongside with the module parameter C(tasks_skip=true).
+    required: false
+    type: list
+    elements: list
```

Comment:
@rainerleber I wonder if it wouldn't be better not to receive a list of parameter-lists, instead make the module execute one single call. Make this arg a `dict`, then specify the `options` (taskname, fieldname, value), with proper types.

For instance, given line 291 below, the module will crash if called like:
```yaml
- name: Pass in input parameters
  community.general.sap_task_list_execute:
    state: present
    conn_username: DDIC
    conn_password: Passwd1234
    host: 10.1.8.10
    sysnr: '00'
    client: '000'
    task_to_execute: SAP_BASIS_SSL_CHECK
    task_parameters :
     - [1234567, 'P_OPT2', 'X']          # <<<<<<<<<<<< number instead of string
     - ['CL_STCT_CHECK_SEC_CRYPTO', 'P_OPT3', 'X']
    task_settings: batch
```